addons/sale/views/sale_views.xml

```diff
@@ -314,6 +314,7 @@
                                             <field name="qty_delivered_manual" invisible="1"/>
                                             <field name="qty_delivered_method" invisible="1"/>
                                             <field name="price_total" invisible="1"/>
+                                            <field name="price_tax" invisible="1"/>
```

Comment:
If I understand correctly, this is necessary for the computation of the total in the onchange, right?

Reply:
It was already forwarded-ported